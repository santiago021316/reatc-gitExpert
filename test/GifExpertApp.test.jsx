import { render ,screen} from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";

describe('pruebas en <GifExpertApp/>>', () => { 


    test('debe hacer el snatShot  de <GifExpertApp/>', () => { 

    const {container} = render(<GifExpertApp/>)
    expect(container).toMatchSnapshot()
    })

   test('debe existir el texto category en el render', () => { 

        render(<GifExpertApp/>)
        expect(screen.getByText('GifExpertApp'))
    })



    test('el length del los heading no debe ser supeior a 2 ', () => { 

      render(<GifExpertApp/>)
       expect(screen.getAllByRole('heading').length).toBe(1)
     })

     test('debe existir una etiqueta form ', () => { 

        render(<GifExpertApp/>)
         expect(screen.getAllByRole('form')).toBeTruthy()
         
       })


 })
 //