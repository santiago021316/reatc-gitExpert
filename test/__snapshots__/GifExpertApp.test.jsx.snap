// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <GifExpertApp/>> debe hacer el snatShot  de <GifExpertApp/> 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="buscar gif"
      type="text"
      value=""
    />
  </form>
  <div />
</div>
`;
